core/lib/config/src/configs/tx_sink.rs

```diff
@@ -0,0 +1,15 @@
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, Clone, PartialEq)]
+pub struct TxSinkConfig {
```

Comment:
Bikeshedding: I don't think this merits a separate config; the only config param could be added to `Web3JsonRpcConfig`.

Reply:
will move it there